test/multiple_config.js

```diff
@@ -0,0 +1,8 @@
+export default function () {
+  return {
+    configFiles: [
+      'test/functional/config.js',
```

Comment:
__question:__ Why can't we use relative paths here, like we would use with `--config` parameter (e.g. `./functional/config.js`)?

Reply:
Good question, was waiting for someone to notice. The path resolution is somewhat of an issue between xpack and kibana, so let me give this another try.